sdk/program/src/serialize_utils/cursor.rs

```diff
@@ -50,6 +53,29 @@ pub(crate) fn read_i64<T: AsRef<[u8]>>(cursor: &mut Cursor<T>) -> Result<i64, In
     Ok(i64::from_le_bytes(buf))
 }
 
+pub(crate) fn read_pubkey_into(
+    cursor: &mut Cursor<&[u8]>,
+    pubkey: *mut Pubkey,
+) -> Result<(), InstructionError> {
+    const PUBKEY_SIZE: usize = mem::size_of::<Pubkey>();
```

Comment:
nit: strictly speaking, i think `solana_sdk::pubkey::PUBKEY_BYTES` is more appropriate than this because this code is dealing with reading N bytes from the buffer. however, `mem::size_of::<Pubkey> == PUBKEY_BYTES` (no padding or whatsoever). and i don't see any possibility of diverting the equality in foreseeable future. so quite minor thing ;)

Reply:
done